minjae/programmers-1/두-개-뽑아서-더하기.js

```diff
@@ -0,0 +1,17 @@
+function solution(numbers) {
+  const set = new Set();
+
+  for (let i = 0; i < numbers.length; i++) {
+    for (let j = i + 1; j < numbers.length; j++) {
+      set.add(numbers[i] + numbers[j]);
+    }
+  }
+  return Array.from(set).sort((a, b) => a - b);
```

Comment:
```suggestion
  return [...set].sort((a, b) => a - b);
```
- Array.from이 아니여도 set은 이터러블이기 때문에 전개 연산자의 피연산자가 될 수 있다. 전개 연산자를 통해서 배열로 변환해서 풀 수도 있다